### ISSUE TITLE
webpush-go and web-push-libs

### ISSUE THREAD

**Issue by gauntface** [SherClockHolmes/webpush-go#4]
Hiya,

I've been looking to move the old [GoogleChrome repo](https://github.com/gauntface/web-push-go) to [web-push-libs org](https://github.com/web-push-libs/), but at the moment it's in a fairly sorry state of affairs and I suspect this library is much further along in terms of web-push support.

Would you be willing to move this repo into web-push-libs? The idea behind the org is to try and get good coverage of web-push in various languages. You'd still be the owner, it would just be easier for people to discover your repo.

On a side note, I'd love to add some integration tests, my Go is pretty poor, but I have some rough tests working in the [integration tests branch here](https://github.com/gauntface/web-push-go/tree/integration-tests). It still needs some work, but it's essentially using [web-push-testing-service](https://github.com/GoogleChrome/web-push-testing-service) to manage the browser set-up. If you are open to a PR for this on your repo, please let me know.

Cheers,
Matt

**Comment by SherClockHolmes** [comment]
@gauntface I like the idea of moving it to the `web-push-libs` org. I have only included VAPID in this implementation so it's currently missing `gcm/fcm` support (_which should be trivial to add_). Feel free to open a PR for your integration tests against this library, then we can start the transfer process.

**Comment by gauntface** [comment]
Woop Woop :tada: 

GCM is the only thing that will require additional work but agree it should be trivial.

Will open PR for integration tests when I get a chance.

**Comment by juan88** [comment]
Hello! Any news on the integration of sending push to chrome?

I've been looking for a single solution for both Chrome and FF but I couldn't manage to do that with @gauntface's lib and this one seems not to be supporting Chrome, right?

**Comment by anaskhan96** [comment]
@juan88 This one supports Chrome just fine. What exactly do you seem to be having a problem with?

**Comment by juan88** [comment]
Ohh maybe I misunderstood something. The thing is I've been using the other lib and I found out that I was receiving unauthorized response codes from FF endpoints. I thought that this was due to problems with VAPID headers and then I found this one. From reading the comments above I thought that the feature of gcm/fcm was related to web push notifications to chrome and that is was not implemented here yet.

Honestly I didn't have the chance to test the code that's why I was asking that. But I insist, maybe I was confused.
